fx/src/test/java/org/unigrid/janus/controller/SettingsControllerTest.java

```diff
@@ -173,4 +178,19 @@ public void shouldChangePassword() {
 		robot.type(KeyCode.ENTER);
 		verifyThat("#pnlWallet", isVisible());
 	}
+
+	@Example
+	public void shouldAskForFile() {
+		new FileChooserMockUp();
+
+		robot.clickOn("#btnSettings");
+		robot.clickOn("#btnSetExport");
+
+		robot.clickOn("#btnSetExportImport");
+		robot.clickOn("#btnSetExportBackup");
```

Comment:
So this needs to be changed to onBackupWallet() as well.

Reply:
do you mean changing to `robot.clickOn("#onBackupWallet");` ?